.gitignore

```diff
@@ -12,59 +12,60 @@ phpstan.neon
 /docs/_build/
 
 # clean up some non bower ready package
-src/Resources/public/vendor/admin-lte/bootstrap
-src/Resources/public/vendor/admin-lte/build
-src/Resources/public/vendor/admin-lte/dist/img
-!src/Resources/public/vendor/admin-lte/dist/img/boxed-bg.jpg # Only image called in AdminLTe.css
-src/Resources/public/vendor/admin-lte/dist/js/app.js
-src/Resources/public/vendor/admin-lte/dist/js/demo.js
-src/Resources/public/vendor/admin-lte/dist/js/pages
-src/Resources/public/vendor/admin-lte/documentation
-src/Resources/public/vendor/admin-lte/pages
-src/Resources/public/vendor/admin-lte/plugins
-src/Resources/public/vendor/admin-lte/*.html
-src/Resources/public/vendor/admin-lte/Gruntfile.js
+Resources/public/vendor/admin-lte/bootstrap
```

Comment:
woops, this looks wrong @greg0ire

Reply:
:thinking: it looks wrong for 3.x too, and for a long time… I don't understand what's happening. I changed this in 54f5e295dd4aafe359287fa4dd542e4411481481, apparently.

Reply:
Can you please answer in my dev-kit PR how we proceed for 3.x and master? And which "version" of the .gitignore file we will keep?
Thanks

Reply:
@core23 shouldn't that move have been done in #4757 ?

Reply:
Yeah, it should have been done in there :/

Reply:
I deployed, 3.x is fixed now, we can close this.